engines/libaio.c

```diff
@@ -335,10 +393,30 @@ static void fio_libaio_cleanup(struct thread_data *td)
 	}
 }
 
+static struct ioengine_ops ioengine = {
+	.name			= "libaio",
+	.version		= FIO_IOOPS_VERSION,
+	.init			= fio_libaio_init,
+	.prep			= fio_libaio_prep,
+	.prio_prep		= NULL,
```

Comment:
[idle speculation] I see what you're going for here (making it generic to other engines) but I can't help wondering if it should be a `libaio` only option for now and it can be done off its queue hook...

Reply:
I'm happy to incorporate the prio selection into the queue hook.